packages/pds/src/api/com/atproto/server/createAccount.ts

```diff
@@ -486,6 +464,42 @@ const ensureUnusedHandleAndEmail = async (
   }
 }
 
+const ensurePhoneVerification = async (
+  ctx: AppContext,
+  phone?: string,
+  code?: string,
+): Promise<string | undefined> => {
+  if (!ctx.cfg.phoneVerification.required || !ctx.twilio) {
+    return
+  }
+
+  if (!phone) {
+    throw new InvalidRequestError(
+      `Text verification is now required on this server. Please make sure you're using the latest version of the Bluesky app.`,
+      'InvalidPhoneVerification',
+    )
+  }
+  if (ctx.cfg.phoneVerification.bypassPhoneNumber === phone) {
+    return undefined
+  }
+
+  if (!code) {
```

Comment:
We might consider trimming the code before this check.

Reply:
nice yup 👍